wordcount/count/FileOpenTextReadLineSearchValuesBenchmark.cs

```diff
@@ -0,0 +1,37 @@
+
+using BenchmarkData;
+
+namespace FileOpenTextReadLineSearchValuesBenchmark;
+
+public static class FileOpenTextReadLineSearchValuesBenchmark
+{
+    public static Count Count(string path)
+    {
+        long wordCount = 0, lineCount = 0, charCount = 0;
+        using StreamReader stream = File.OpenText(path);
+
+        string? line = null;
+        while ((line = stream.ReadLine()) is not null)
+        {
+            lineCount++;
+            charCount += line.Length;
+            ReadOnlySpan<char> text = line.TrimStart();
```

Comment:
This is a potential allocation.

Reply:
Oops. That doesn't do what I thought it does.

Reply:
`ReadOnlySpan<char> text = line.AsSpan().TrimStart();`